### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by TelefunkenU87**
Completed lesson 3 homework assignment. Refactored code from one block into multiple functions.

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896520)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896520&bid=16215794)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896520)

File: Src/BootCamp.Chapter/Checks.cs [hunk 3]
```diff
@@ -16,26 +16,22 @@ public static class Checks
...
-            // To do: call your implementation. 
-            return 0;
+            return Lesson3.PromptFloat(message);
         }
 
         public static float CalculateBmi(float weight, float height)
         {
-            // To do: call your implementation. 
-            return 0;
+            return Lesson3.CalculateBmi(weight, height);
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Checks.cs]: The value of height is in meter here. Adjust it so you can use it for your function correctly. :)

**Comment by TelefunkenU87** [comment]
I changed the prompt and the formula so that it is expecting meters instead of centimeters. 

I also looked into string interpolation more since homework 1+2 and our last review, Ellmaster. I think I have a better understanding of what you were trying to recommend. Doing my first pull request and the dilemma I was having with running code without opening the solution was too much for this poor little programmer in training at the time. I appreciate you pointing it out at the time.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896520&bid=16215794)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896520)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 2 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896520)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896520&bid=16215794)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+            string lName = PromptString("Enter last name:");
+
+            int age = PromptInt("Enter age:");
+
+            float weight = PromptFloat("Enter wight in kg:");
+
+            float height = PromptFloat("Enter height in meters:");
+
+            Console.WriteLine($"{fName} {lName} is {age} years old, their weight is {weight} kg and their height is {height} m.");
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Lesson3.cs]: Very good! :)

**Pull request closed without merging**